packages/components/src/Tag/Tag.tsx

```diff
@@ -0,0 +1,69 @@
+// @flow
+
+import * as React from "react";
+import Text from "../Text";
+import { TypographyColor } from "../common/typography";
+import cx from "classnames";
+import styles from "./Tag.module.css";
+
+export type Color = Omit<
+  TypographyColor,
+  "white" | "base" | "info" | "inherit"
+>;
+
+export const stylesByColor: { [K in Color]: string } = {
```

Comment:
I think we no longer need the `{ |K in Color]: string }` annotation, typescript can now infer it (unlike flow)